README.md

```diff
@@ -710,18 +739,21 @@ Offline Installations   | ✓     | x   | x   | x    |
 
 ### Current Tool Versions
 
-Tool   | Version |
----    |---------|
-Go     | 1.18.6  |
-Npm    | 8.3.1   |
-Node   | 16.14.0 |
-Pip    | 21.3.1  |
-Python | 3.10.6  |
-Git    | 2.37.3  |
-Yarn*  | 1.x     |
+Tool      | Version |
+---       |---------|
+Go        | 1.18.6  |
+Npm       | 8.3.1   |
+Node      | 16.14.0 |
+Pip       | 21.3.1  |
+Python    | 3.10.6  |
+Git       | 2.37.3  |
+Yarn*     | 1.x     |
+RubyGems* | 2.x     |
```

Comment:
Perhaps the docs should mention Bundler (as the package manager) instead of "RubyGems" ?

(or else, I'm not sure what `2.x` means in the context of "RubyGems" ? )